lib/game.rb

```diff
@@ -0,0 +1,71 @@
+class Game
+  attr_reader :current_turn, :turn_count, :turn_return
+  def initialize(current_turn)
+    @current_turn = current_turn
+    @turn_count = 1
+    @turn_return = nil
+  end
+
+  # Determine if a player has 0 cards in their deck
+  def the_end?
+    if @current_turn.player_one.has_lost? || @current_turn.player_two.has_lost?
+      true
+    end
+  end
```

Comment:
So this predicate method, `the_end?`, only returns `true` or `nil`? What happened to `false`? Remember, in Ruby, it is customary that if a method name ends with a `?`, then it is assumed that it will return either `true` or `false`. We want to make sure to stick with conventions so other Ruby developers can make assumptions of our code and interface with it properly.